x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/client/CcrClient.java

```diff
@@ -96,6 +98,16 @@ public void unfollow(final UnfollowAction.Request request, final ActionListener<
         return listener;
     }
 
+    public void forgetFollower(final ForgetFollowerAction.Request request, final ActionListener<BroadcastResponse> listener) {
+        client.execute(ForgetFollowerAction.INSTANCE, request, listener);
+    }
+
+    public ActionFuture<BroadcastResponse> forgetFollower(final UnfollowAction.Request request) {
```

Comment:
should be `ForgetFollowerAction.Request` instead of `UnfollowAction.Request`

Reply:
It makes me sad we do not have the compiler catching that anymore. Good catch!

Reply:
I pushed 33edd224e758b0a816d58083213fc676a5eb6d5d.